### DISCUSSION TITLE
Docs showing v5/v6

### DISCUSSION THREAD

**Discussion by LucidNinja** [react-hook-form/react-hook-form#9631] [General] [answered]
I'm not sure if this is intentional, but react-hook-form.com shows the v5/v6 docs by default, and then when switching to v7, it goes to legacy.react-hook-form.com. Is this intentional?

**Comment by Moshyfawn** [comment, marked as answer]
If you mean the `5 / 6` link that's shown by default when you hit https://react-hook-form.com/, that's intended. You get v7 when you land on the main page. If you click `5 / 6`, you get redirected to https://legacy.react-hook-form.com/ where you can switch between v5 and v6 _(won't be reflected in the URL)_

**Reply by LucidNinja** [reply]
Oh, it's a link - maybe that's a little unclear. Lots of docs show the current version, not a different version. I interpreted that as if I'm looking at Version 5/6 docs. To clarify, if I'm on react-hook-form.com, that is showing Version 7?

**Reply by Moshyfawn** [reply]
I can see how it can be confusing. I was working on updating the docs, this part included, but life got in the way. I'll make sure to improve the versioning part as well, when I get to it!

Yes, https://react-hook-form.com/ - the v7 docs

**Reply by LucidNinja** [reply]
Great! Thanks!
